src/Core/Models/Data/EmergencyAccessDetails.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+using System.Collections.Generic;
+using Bit.Core.Enums;
+using Bit.Core.Models.Table;
+using Newtonsoft.Json;
+
+namespace Bit.Core.Models.Data
+{
+    public class EmergencyAccessDetails : EmergencyAccess, ITwoFactorProvidersUser
+    {
+        private Dictionary<TwoFactorProviderType, TwoFactorProvider> _twoFactorProviders;
+
+        public string GranteeName { get; set; }
+        public string GranteeEmail { get; set; }
+        public string GranteeTwoFactorProviders { get; set; }
+        public bool? GranteePremium { get; set; }
+        public string GrantorName { get; set; }
+        public string GrantorEmail { get; set; }
+
+        public string TwoFactorProviders
+        {
+            get { return GranteeTwoFactorProviders; }
+        }
```

Comment:
Not sure I understand the purpose of the property, but it could be shorthanded like this:

```suggestion
        public string TwoFactorProviders => GranteeTwoFactorProviders;
```